### PR TITLE
Fix #3  : Add Code Owner Template

### LINKED ISSUE #3: Codeowner template
Add Codeowner template
Mention @rt4914 as codeowner for now.

References:
https://github.com/forem/forem/blob/main/.github/CODEOWNERS
https://github.com/oppia/oppia-android/blob/develop/.github/CODEOWNERS

### PR DESCRIPTION
**Pull request by rray524**
Fix #3  : Add code owner Template

### Description

This pull request introduces a new standardized template for `CODEOWNERS` files. This template ensures that ownership of code areas is clearly defined and that all necessary information is included for maintaining and reviewing code contributions. This change will help streamline the code review process by automatically assigning the right people to review specific parts of the codebase.

### Type of Change

* Added a standardized `CODEOWNERS` template to the repository.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (rt4914) [inline .github/CODEOWNERS:5]: Remove everything below this.

File: .github/CODEOWNERS
Comment (rt4914) [inline .github/CODEOWNERS:3]: This should be 

* @rt4914

**Review by rt4914 — CHANGES_REQUESTED** [review]
Also, I think you made a mistake while creating this branch.

You should have created this new branch from `main` branch. Because of this mistake there are changes in this PR which are related to `ISSUE_TEMPLATE` which is a separate PR.

So for now what you can do is wait for the other PR to get merged, once that PR gets merged you can come back to this branch and make it up-to-date with latest main branch. That way you won't see any other files and only CODEOWNERS file will be there.

If this is not clear feel free to reach out.

**Comment by rray524** [comment]
> Also, I think you made a mistake while creating this branch.
> 
> You should have created this new branch from `main` branch. Because of this mistake there are changes in this PR which are related to `ISSUE_TEMPLATE` which is a separate PR.
> 
> So for now what you can do is wait for the other PR to get merged, once that PR gets merged you can come back to this branch and make it up-to-date with latest main branch. That way you won't see any other files and only CODEOWNERS file will be there.
> 
> If this is not clear feel free to reach out.

thanks for your clearification. sorry about it. @rt4914

File: .github/CODEOWNERS
Comment (rray524) [reply .github/CODEOWNERS:5]: removed. thanks

File: .github/CODEOWNERS
Comment (rray524) [reply .github/CODEOWNERS]: I have changed to above requirement. Thanks

File: .github/CODEOWNERS
Comment (rt4914) [reply .github/CODEOWNERS:5]: This last line is still visible. Also line 3 there is some sort of dot visible. it should be star.
Overall file should be:
```
# Global owner for everything in this repo

* @rt4914

```

**Review by rt4914 — CHANGES_REQUESTED** [review]
@rray524 PTAL.

File: .github/CODEOWNERS
Comment (rray524) [reply .github/CODEOWNERS]: Resolved. the issue. please check now. thanks. @rt4914

File: .github/CODEOWNERS
Comment (rt4914) [inline .github/CODEOWNERS:3]: This one is still incorrect.

It should 
`* @rt4914` currently its missing the * at the start.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@rray524 Please take a look. The last changes were not correct.

**Comment by rt4914** [comment]
@rray524 you will need to create a new PR. We cannot merge a `force-pushed` PR as it tampers the entire commit history. 

Closing this PR. You will need to create a new one.

File: .github/CODEOWNERS
Comment (rt4914) [reply .github/CODEOWNERS]: Okay we will merge this, but if it creates an issue we will need to reopen the issue and fix this again.

**Pull request closed without merging**
